spec/sales_engine_spec.rb

```diff
@@ -37,11 +37,11 @@
     end
 
     it 'has Items in the ItemRepository' do
-      item_details = MockData.items_as_hash
-      mock_items = MockData.items_as_mocks(item_details) { self }
+      item_details = MockData.items_as_hashes
+      mock_items = MockData.items_as_mocks(self, item_details)
```

Comment:
Similarly, is there a reason you're using the longer syntax here? Maybe I'm missing something about how this is different than the way you used it in the ItemRepo tests

Reply:
No this is a good catch, let me make that change right quick